requirements/base.in

```diff
@@ -1 +1,2 @@
 Django ~= 3.0
+coverage==7.3.2
```

Comment:
Let's not add test-only dependencies to the base requirements. Use a separate requirements file for non-production envs. Also, no need to pin the version in this file because it's an input file and the pins need to be updateable by pip-tools in the future, don't prevent this.

Reply:
ок

Reply:
Marking this as resolved.